horace_core/sqw/PixelData/@MultipixBase/private/set_npix_each_file_.m

```diff
@@ -0,0 +1,17 @@
+function obj = set_npix_each_file_(obj,val)
+%SET_NPIX_EACH_FILE_ main setter of npix property.
+% Sets the numeric array which defines number of pixels in each file or
+% signle value if total number of pixels in each file is the same
+if ~isnumeric(val)
+    error('HORACE:pixfile_combine_info:invalid_argument',...
+        'pos_npixstart has to be numeric array containing information about npix location on hdd')
```

Comment:
Prefer "in file" as "on hdd" may not strictly be correct.
```suggestion
        'pos_npixstart has to be numeric array containing information about npix location in file')
```

Reply:
That's all wrong (from different comment)